core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerImpl.java

```diff
@@ -264,11 +266,31 @@ public void selectAudioTrack(PlayerAudioTrack audioTrack) {
     }
 
     @Override
+    public void showSubtitleTrack(PlayerSubtitleTrack subtitleTrack) {
+        NoPlayerLog.w("Subtitles not implemented for Android Media Player");
+    }
+
+    @Override
+    public void showFirstAvailableSubtitleTrack() {
+        NoPlayerLog.w("Subtitles not implemented for Android Media Player");
+    }
+
+    @Override
+    public void hideSubtitleTrack() {
+        NoPlayerLog.w("Subtitles not implemented for Android Media Player");
+    }
+
+    @Override
     public List<PlayerAudioTrack> getAudioTracks() {
         return mediaPlayer.getAudioTracks();
     }
 
     @Override
+    public List<PlayerSubtitleTrack> getSubtitleTracks() {
+        return mediaPlayer.getSubtitleTracks();
```

Comment:
Why does this one delegate to a non implementation when the other methods are handled here ?